src/main/java/com/minecolonies/coremod/entity/ai/citizen/smelter/EntityAIWorkSmelter.java

```diff
@@ -34,7 +34,7 @@
  */
 public class EntityAIWorkSmelter extends AbstractEntityAISkill<JobSmelter>
 {
-    //todo add status for sign over their head
+    //TODO: add status for sign over their head
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)